sdk/core/core-amqp/package.json

```
@@ -10,47 +10,32 @@
     "amqp",
     "cloud"
   ],
-  "main": "./dist/index.js",
-  "module": "./dist-esm/src/index.js",
-  "types": "./types/latest/core-amqp.d.ts",
-  "browser": {
-    "./dist-esm/src/util/checkNetworkConnection.js": "./dist-esm/src/util/checkNetworkConnection.browser.js",
-    "./dist-esm/src/util/hmacSha256.js": "./dist-esm/src/util/hmacSha256.browser.js",
-    "./dist-esm/src/util/runtimeInfo.js": "./dist-esm/src/util/runtimeInfo.browser.js",
-    "buffer": "buffer"
-  },
-  "react-native": {
-    "./dist/index.js": "dist-esm/src/index.js",
-    "./dist-esm/src/util/runtimeInfo.js": "./dist-esm/src/util/runtimeInfo.native.js"
-  },
   "files": [
     "dist/",
-    "dist-esm/src/",
-    "types/latest/core-amqp.d.ts",
     "README.md",
     "LICENSE"
   ],
   "scripts": {
-    "bundle": "tsc -p . && dev-tool run bundle --polyfill-node false --inject-node-polyfills true --ignore-missing-node-builtins true",
+    "bundle": "tshy && dev-tool run bundle --polyfill-node false --inject-node-polyfills true --ignore-missing-node-builtins true",
     "build:samples": "echo Obsolete",
     "build:test": "echo skipped. actual commands inlined in browser test scripts",
-    "build": "npm run clean && tsc -p . && dev-tool run bundle && dev-tool run extract-api",
+    "build": "npm run clean && tshy &&  dev-tool run build-test && dev-tool run extract-api",
```

probably don't need to build test here.
```suggestion
    "build": "npm run clean && tshy && dev-tool run extract-api",
```